scripts/update_apkid.sh

```diff
@@ -0,0 +1,28 @@
+script_path=$(dirname $0)
+if [ "$script_path" != "scripts" ] && [ "$script_path" != "./scripts" ]; then
+    echo 'Please run script from MobSF directory '
+    echo './scripts/update_apkid.sh '
+    exit  1
+fi
+virtualenv venv -p python3
+source venv/bin/activate
+virtual_env=$(echo $VIRTUAL_ENV)
+if [ -e "$virtual_env" ]; then
+    apkid_dir="${virtual_env}/lib/python3.6/site-packages/apkid"
+    if [ ! -d "$apkid_dir" ];then
+      pip install apkid
```

Comment:
The rednaga fork of yara-python needs to be installed first right? If we are having script for updating APKiD rules, then why not one for installing yara-python ?